app/editor/src/features/content/form/constants/clipDirectoryColumns.tsx

```diff
@@ -0,0 +1,86 @@
+import { IItemModel } from 'hooks/api-editor';
+import {
+  FaCloudDownloadAlt,
+  FaPaperclip,
+  FaPhotoVideo,
+  FaPlay,
+  FaRegFolder,
+  FaTrash,
+} from 'react-icons/fa';
+import { Column } from 'react-table';
+
+/** columns located within file for state manipulation */
+export const clipDirectoryColumns = (
+  deleteItem: Function,
```

Comment:
I would rename to follow event based naming practices; onDelete, onSelect, onDownload, onAttach.

Reply:
I was using the naming in tno\app\editor\src\features\storage\StorageListView.tsx as a style guide. Should I change the naming convention in both of these?

Reply:
I would change them if they're events that occur and are being bubbled out to the calling component.

Reply:
Fixed.